src/sagas/wallet.js

```diff
@@ -488,9 +526,9 @@ export function* handleTx(action) {
     return acc;
   }, [{}, new Set([])],);
 
-  const txWalletAddresses = yield call(wallet.checkAddressesMine.bind(wallet), [...txAddresses]);
-  const tokensToDownload = [];
+  const txWalletAddresses = yield call(checkAddressMine, wallet, [...txAddresses]);
```

Comment:
Instead of only retrying the first http request, why don't we retry the entire handleTx?

Reply:
I don't see why to retry the entire handle effect because this is an issue of the client that don't handle a possible network error.

Reply:
A suggestion here would be to abstract this further and create a generic "retry saga" in helpers which receives a promise and a number of retries as a parameter and retries that promise until the max number of retries is reached

What I don't like here is adding an abstraction to this specific method just because it is the first http request in this saga

What do you think?

Reply:
Humm... It is good for me. I will implement this helper.

Reply:
Done at https://github.com/HathorNetwork/hathor-wallet-mobile/pull/444/commits/dbbb4582a0ea6d03c5969b26ee5cb86421d906ae